modules/local/starfusion/download/main.nf

```diff
@@ -38,10 +31,10 @@ process STARFUSION_DOWNLOAD {
     wget https://www.dfam.org/releases/Dfam_3.4/infrastructure/dfamscan/homo_sapiens_dfam.hmm.h3i
     wget https://www.dfam.org/releases/Dfam_3.4/infrastructure/dfamscan/homo_sapiens_dfam.hmm.h3m
     wget https://www.dfam.org/releases/Dfam_3.4/infrastructure/dfamscan/homo_sapiens_dfam.hmm.h3p
-    
+
     gunzip Pfam-A.hmm.gz && hmmpress Pfam-A.hmm
 
-    prep_genome_lib.pl \\
+    /usr/local/src/STAR-Fusion/ctat-genome-lib-builder/prep_genome_lib.pl \\
```

Comment:
This will break if you use conda. Maybe there should be some check for the environment. Something on the line of
```groovy
def binPath = if (workflow.containerEngine == "singularity" || workflow.containerEngine == "docker") ? "/usr/local/src/STAR-Fusion/ctat-genome-lib-builder" : ""

$binPath/prep_genome.pl ...
```

Reply:
Very good point!